src/TBuffer.h

```diff
@@ -95,7 +95,9 @@ struct TMxpElement
 
 class TBuffer
 {
-    static const QMap<QString, QVector<QChar>> csmEncodingTable; // private!
```

Comment:
Well, not exactly, it does not allow passing disambiguation text to the translator, and while Qt documentation says this is [no longer the preferred](http://doc.qt.io/qt-5/i18n-source-translation.html#disambiguation) way of doing so, it fails to provide the modern way.